documentation/book/con-kafka-mirror-maker-authentication.adoc

```diff
@@ -1,98 +1,55 @@
 
-To use TLS client authentication, set the `type` property to the value `tls`.
 TLS client authentication uses a TLS certificate to authenticate.
 The certificate is specified in the `certificateAndKey` property and is always loaded from a Kubernetes secret.
 In the secret, the certificate must be stored in X509 format under two different keys: public and private.
 
 NOTE: TLS client authentication can be used only with TLS connections.
-For more details about TLS configuration in Kafka Mirror Maker see xref:assembly-kafka-mirror-maker-tls-{context}[].
-
-.An example TLS client authentication configuration
-[source,yaml,subs=attributes+]
-----
-apiVersion: {KafkaApiVersion}
-kind: KafkaMirrorMaker
-metadata:
-  name: my-mirror-maker
-spec:
-  # ...
-  consumer:
-    authentication:
-      type: tls
-      certificateAndKey:
-        secretName: my-source-secret
-        certificate: public.crt
-        key: private.key
-  # ...
-  producer:
-    authentication:
-      type: tls
-      certificateAndKey:
-        secretName: my-target-secret
-        certificate: public.crt
-        key: private.key
-  # ...
-----
-
-== SCRAM-SHA-512 authentication
-
-To configure Kafka Mirror Maker to use SCRAM-SHA-512 authentication, set the `type` property to `scram-sha-512`.
-The broker listener to which clients will connect must also be configured to use SCRAM-SHA-512 SASL authentication.
+
+For more information on configuring TLS encryption and authentication, see:
+
+* xref:proc-configuring-kafka-mirror-maker-tls-{context}[Configuring TLS encryption in Kafka Mirror Maker]
+* xref:proc-configuring-kafka-mirror-maker-authentication-tls-{context}[Configuring TLS client authentication in Kafka Mirror Maker]
+
+.SCRAM-SHA-512 authentication
+
+Use `type.scram-sha-512` to configure Kafka Mirror Maker to use SCRAM-SHA-512 authentication.
+
+The broker listener to which clients will connect must also be configured to use SASL SCRAM-SHA-512 authentication.
 This authentication mechanism requires a username and password.
 
 * Specify the username in the `username` property.
-* In the `passwordSecret` property, specify a link to a `Secret` containing the password. The `secretName` property contains the name of the `Secret` and the `password` property contains the name of the key under which the password is stored inside the `Secret`.
+* In the `passwordSecret` property, specify a link to a `Secret` containing the password.
+The `secretName` property contains the name of the `Secret` and the `password` property contains the name of the key under which the password is stored inside the `Secret`.
 
-IMPORTANT: Do not specify the actual password in the `password` field.
+For more information on configuring SASL SCRAM-SHA-512 authentication, see xref:proc-configuring-kafka-mirror-maker-authentication-scram-sha-512-{context}[Configuring SCRAM-SHA-512 authentication in Kafka Mirror Maker].
+
+.PLAIN authentication
```

Comment:
Could we keep this as a heading? It is quite important because it allows us to link to the specific section.

```suggestion
=== PLAIN authentication
```